src/Ryujinx.Headless.SDL2/Program.cs

```diff
@@ -439,6 +440,8 @@ void LoadPlayerConfiguration(string inputProfileName, string inputId, PlayerInde
             GraphicsConfig.ShadersDumpPath = option.GraphicsShadersDumpPath;
             GraphicsConfig.EnableMacroHLE = !option.DisableMacroHLE;
 
+            StartFullscreen = option.StartFullscreen;
```

Comment:
Why do not set a field in `WindowBase` instead ?

```suggestion
            _window.IsFullscreen = option.IsFullscreen;
```